app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -282,16 +274,29 @@ public void onConnectionSuspended(int i) {
     @Override
     public void onConnectionFailed(@NonNull ConnectionResult result) {
         if (mSignInProgress != STATE_IN_PROGRESS) {
+            if (!GoogleApiAvailability.getInstance().isUserResolvableError(result.getErrorCode())) {
+                if (!PrefUtils.shouldShowFatalPlayServiceMessage(this)) {
+                    return;
+                }
+                PrefUtils.setFatalPlayServiceMessageShown(this);
+            }
+
             // We do not have an intent in progress so we should store the latest
             // error resolution intent for use when the sign in button is clicked.
             mSignInIntent = result.getResolution();
-
-            //if (mSignInProgress == STATE_SIGN_IN) {
-            // STATE_SIGN_IN indicates the user already clicked the sign in button
-            // so we should continue processing errors until the user is signed in
-            // or they click cancel.
-            resolveSignInError();
-            //}
+            if (mSignInIntent != null) {
+                resolveSignInError();
+            } else {
+                // Google Play services wasn't able to provide an intent for some
+                // error types, so we show the default Google Play services error
+                // dialog which may still start an intent on our behalf if the
+                // user can resolve the issue.
+                if (PrefUtils.shouldShowFatalPlayServiceMessage(this)) {
+                    Timber.e("Google Play Service did not resolve error");
+                    GoogleApiAvailability.getInstance().showErrorNotification(this, result.getErrorCode());
```

Comment:
Makes sense :+1:

It was just a personal thing. I hate that dialog. I am seeing it everywhere. It didn't even help me once. :☺